packages/devtools_app/test/shared/analytics_prompt_test.dart

```diff
@@ -11,13 +11,15 @@ import 'package:devtools_test/helpers.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:provider/provider.dart';
+import 'package:unified_analytics/src/constants.dart' as unified_analytics;
```

Comment:
@kenzieschmoll is this okay? We aren't explicitly adding the package into the pubspec for `devtools_app` but I was wondering if it would be okay i pulled in the constant like this?

Reply:
Can we add this as a dependency in the pubspec? Won't we need to access the constants anyway as we migrate all of our devtools events to package:unified_analytics?

Reply:
oh right! i'll add it in